lips/lip-10.md

```diff
@@ -0,0 +1,231 @@
+* Only special Treasury Compliance account (address `0xB1E55ED`) can manipulate LibraIdDomains resource:
+  * Only TC account can create and publish LibraIdDomains resource
+  * Only TC account can add, remove or update an LibraIdDomain within LibraIdDomains resource
+* In order to register a Libra Id domain, VASP needs to submit a request to Libra Association. LA will perform certain checks (out of scope of this document) before they will issue an on-chain transaction to register a Libra Id Domain. The checks that LA will perform will make sure that VASPs do not claim irrelevant Libra Id Domains. LA submission will also ensure that Libra Id Domains are unique.
+* It should also be possible (and probably going to be most common) for a new VASP to submit their desired Libra ID domain(s) along with the request to create a `ParentVASP` account.
+* Association may provide an open source application(*Indexer*) that will allow indexing available Libra IDs and will provide a convenient REST api for applications to fetch information about the domains. The API of such an indexer is out of scope of this RFC.
+
+# Peer-2-peer payment protocol
+One of the main use cases for Libra ID is transferring money from one user to another across different wallets.
+
+<img alt="Payment flow" src={useBaseUrl('img/libra-id-flow.png')} />
+
+## UI flow
+* Receiver shares their Libra ID with sender
+* Sender enters receiver’s Libra ID(and other txn details such as amount) in their wallet app
+* Sender’s wallet can fetch profile picture and full name of the receiver prior to sending money, and show it to sender, to improve security (if receiver VASP allows it)
+* Sender presses ‘send’ button
+* Receiver receives money and information about the sender (optionally)
+
+## Backend flow
+In order to support peer-2-peer payments, VASPs need to implement two Libra ID endpoints:
+
+
+* **Info endpoint** for querying optional information about the receiver. VASPs can provide this endpoint to improve UX and security of payment, but they are not obligated to - VASPs may choose not to share any information about the user via this endpoint, including whether the user exists or not.
+  * *For example*: VASP can have notion of 'contact list' and only share information if the sender is in receiver's contacts
+* **Payment endpoint** to perform actual money transfer.
+
+Common considerations:
+
+* For both endpoints, all communications are signed via the same mechanism as for other off-chain communications, see the [Payloads section of lip-1](https://github.com/libra/lip/blob/master/lips/lip-1.mdx#payloads) for details. As such, all vasp-2-vasp communications are authenticated.
+
```

Comment:
I think a lot of this and down makes it seem like this is separate but similar to LIP-1, but to me it makes more sense to state it as an extension of LIP-1 that adds new functionality (basically a query API and a new command in the normal API).  Then you don't really need to re-state the things of LIP-1 which I think kind of add confusion since we only state some of the things